linux_os/guide/system/software/integrity/fips/grub2_enable_fips_mode/anaconda/shared.anaconda

```diff
@@ -1,3 +1,3 @@
-# platform = Red Hat Enterprise Linux 7,Red Hat Enterprise Linux 8,multi_platform_fedora
+# platform = Red Hat Enterprise Linux 7
```

Comment:
Still applicable to rhel8. Please undo the changes.

Reply:
@redhatrises  I took a look into this again and this Anaconda remediation only adds "dracut-fips" package which again doesn't exist in RHEL8.

Reply:
I consider this solved, there is not package `dracut-fips` in RHEL8.